Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:18.04
+
+COPY . /usr/src/
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get update -qq && apt-get install -y \
+    build-essential \
+    curl \
+    clang \
+    cmake \
+    git \
+    g++ \
+    nodejs \
+    libssl-dev \
+    llvm \
+    netcat \
+    pkg-config \
+    python3 \
+    && rm -rf /var/lib/apt/lists/*
+
+
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH
+
+RUN curl https://sh.rustup.rs -sSf | \
+    sh -s -- -y --no-modify-path --default-toolchain nightly-2020-05-15
+
+
+COPY config* ~/.rainbowup
```

Comment:
I think config should be mount as volume, instead of baked into docker image. so people don't rebuild image for config changes

Reply:
This files is needed in that repo which gets used inside the image, even if we make this a volume we would need manual copy for the config inside the container. We can discuss this in more detail.

Reply:
No, if you mount config* as volume ~/.rainbowup, from container's perspective config* file is a file inside container, you don't need to copy that.

Reply:
Cool, that's true. We need to change the prepare script to copy the config file to ~/.rainbowup in that case first and than we can just mount it there. That makes sense. Do you mind if I do it as a follow up issue?

Reply:
Sure!

Reply:
Created an issue for this: https://github.com/near/rainbow-bridge/issues/92.